java/src/com/ibm/streamsx/topology/spl/SubmissionParameter.java

```diff
@@ -0,0 +1,26 @@
+package com.ibm.streamsx.topology.spl;
+
+import com.ibm.streamsx.topology.builder.BSubmissionParameter;
+
+public class SubmissionParameter extends BSubmissionParameter {
```

Comment:
The code started out that way :-)   The issue was that topology.builder.BOperatorInvocation.setParameter() would then end up with a dependency on SubmissionParameter, which seemed best avoided since topology.builder seemed to have virtually no other dependencies on other streamsx.topology entities, or that callers of BOperatorInvocation(...,params) would need to first massage the params to cvt to a BSubmissionParameter, which seemed unnecessarily cumbersome/error prone.  I could convert BSubmissionParameter to an interface though I don't think that fundamentally changes things.  But I'll make the change to that you suggested if that's your preference.  Let me know.